Automattic/wp-calypso
# Build: Fix display issues in Calypso ascii art

This pull request fixes few display issues in the ascii art displayed in the console when starting Calypso and in the HTML source of each page:

##### Before

```
vagrant@calypso:/var/sources$ make run
    ___ __ _| |_   _ _ __  ___  ___
  / __/ _\` | | | | | '_ \/ __|/ _ \
  | (_| (_| | | |_| | |_) \__ \ (_) |
   \___\__,_|_|\__, | .__/|___/\___/
               |___/|_|

Hash: 81b6ed3ffaa88e373253
Version: webpack 1.12.6

...
```

##### After

```
vagrant@calypso:/var/sources$ make run
             _
    ___ __ _| |_   _ _ __  ___  ___
   / __/ _` | | | | | '_ \/ __|/ _ \
  | (_| (_| | | |_| | |_) \__ \ (_) |
   \___\__,_|_|\__, | .__/|___/\___/
               |___/|_|

Hash: 81b6ed3ffaa88e373253
Version: webpack 1.12.6

...
```

#### Testing instructions

1. Execute `make run` and check the welcome message displayed in the console
2. Navigate to any page of Calypso and check the banner displayed in the header of the HTML source

/cc @nb, @folletto
